custom/icds_reports/tests/test_agg_skipping.py

```diff
@@ -0,0 +1,96 @@
+import uuid
+from unittest.mock import patch
+
+from django.core import management
+from django.test import TestCase
+
+from corehq.util.test_utils import generate_cases
+from custom.icds_reports.models.util import AggregationRecord
+
+
+class TestPreviousMonthSkipping(TestCase):
+    def _test_prev_month(self, date_string, prev_month_called):
+        prev_month_agg_uuid = uuid.uuid4().hex
+
+        # verify that setup_aggregation is only run when previous month should be run
+        with patch(
+                'custom.icds_reports.management.commands.create_aggregation_record.setup_aggregation',
```

Comment:
These tests are now using the management commands directly instead of just testing a utility.